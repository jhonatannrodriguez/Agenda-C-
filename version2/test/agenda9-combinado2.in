crearAgenda

estaEnAgenda 1

crearFecha 19/4/2023
crearEvento 1 Descripcion uno
agregarEnAgenda

estaEnAgenda 1

crearFecha 15/4/2023
crearEvento 2 Descripcion dos
agregarEnAgenda

crearFecha 21/4/2023
crearEvento 3 Descripcion tres
agregarEnAgenda

crearFecha 31/12/2022
crearEvento 4 Descripcion cuatro
agregarEnAgenda

imprimirAgenda

posponerEnAgenda 4 109

imprimirAgenda

hayEventosFechaAgenda 31/12/2022

imprimirEventosFechaAgenda 19/4/2023

removerDeAgenda 1

imprimirCopiaAgenda

liberarAgenda

Fin
