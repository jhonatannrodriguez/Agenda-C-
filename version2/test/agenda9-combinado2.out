1>2>El evento con id 1 NO está en la agenda.
3>4>5>6>El evento con id 1 está en la agenda.
7>8>9>10>11>12>13>14>15>16>Evento 4:  Descripcion cuatro
Fecha: 31/12/2022
Evento 2:  Descripcion dos
Fecha: 15/4/2023
Evento 1:  Descripcion uno
Fecha: 19/4/2023
Evento 3:  Descripcion tres
Fecha: 21/4/2023
17>La nueva fecha del evento 4 pospuesto 109 dias es: 19/4/2023
18>Evento 2:  Descripcion dos
Fecha: 15/4/2023
Evento 4:  Descripcion cuatro
Fecha: 19/4/2023
Evento 1:  Descripcion uno
Fecha: 19/4/2023
Evento 3:  Descripcion tres
Fecha: 21/4/2023
19>No se encontraron eventos en la fecha determinada.
20>Evento 4:  Descripcion cuatro
Fecha: 19/4/2023
Evento 1:  Descripcion uno
Fecha: 19/4/2023
21>El evento con id 1 se removió de la agenda.
22>Evento 2:  Descripcion dos
Fecha: 15/4/2023
Evento 4:  Descripcion cuatro
Fecha: 19/4/2023
Evento 3:  Descripcion tres
Fecha: 21/4/2023
23>24>Fin.
